ui/imports/shared/views/chat/MessageView.qml

```diff
@@ -204,6 +204,8 @@ Loader {
             return fetchMoreMessagesButtonComponent
         case Constants.messageContentType.systemMessagePrivateGroupType:
             return privateGroupHeaderComponent
+        case Constants.messageContentType.systemMessagePinnedMessage:
+            return null
```

Comment:
Do you think it was crashing because we were trying to create a `messageCompoennt` when it was actually a `systemMessagePinnedMessage`?

If that's the case, we'll have crashes each time a new MessageType is added.

So we should make the `default` `return null`, because then we just show nothing because we didn't implement it yet, and we need to explicitly list the types that **should** use `messageComponent`.

Reply:
It was crashing on the Nim side in the code like `message.contentType.ContentType`. I.e. when trying to convert integer to `enum`. 😐

I agree that it would be great fix that on another level, but thought that it's not easy to implement.
But didn't check that. Let me take a look 🙂